### PR TITLE
PXC-3822 Improve compression on stream section of xtrabackup_sst (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/manual/xtrabackup_sst.rst

### CODE REVIEW

File: doc/source/manual/xtrabackup_sst.rst [hunk 1]
```diff
@@ -213,28 +213,35 @@ For example: ::
...
     :Default: not set (disabled)
-    :Example: decompressor='gzip -dc'
+    :Example: decompressor='zstd -TO -dc'
 
-Two previous options enable stream-based compression/decompression.
-When these options are set, compression/decompression is performed on stream,
-in contrast to performing decompression after streaming to disk,
-involving additional I/O.
-This saves a lot of I/O (up to twice less I/O on joiner node).
+Stream-based compression and decompression is performed on the stream, in contrast to performing decompression after streaming to disk, which involves additional I/O. The savings are considerable, up to half the I/O on the JOINER node.
```
Comment (alina-derkach-oaza) [inline doc/source/manual/xtrabackup_sst.rst]: is performed -> are performed

File: doc/source/manual/xtrabackup_sst.rst [hunk 2]
```diff
@@ -213,28 +213,35 @@ For example: ::
...
+You must install the related binaries, otherwise SST aborts.
+
+compressor='pigz'
+decompressor='pigz -dc'
+
+compressor='gzip'
+decompressor='gzip -dc'
+
+To revert to the XtraBackup-based compression, 
+set ``compress`` under ``[xtrabackup]``. You can have both compressor and decompressor defined although you will be wasting CPU cycles.
```
Comment (alina-derkach-oaza) [inline doc/source/manual/xtrabackup_sst.rst]: You can have both compressor and decompressor defined - add a comma after "defined"

File: doc/source/manual/xtrabackup_sst.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/manual/xtrabackup_sst.rst]: done

File: doc/source/manual/xtrabackup_sst.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/manual/xtrabackup_sst.rst]: rewritten

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**